doc/requirements.txt

```diff
@@ -2,5 +2,6 @@ sphinx==2.4.4
 breathe==4.13.0
 exhale
 sphinx_rtd_theme
-recommonmark
+myst-parser==0.14.0a3
```

Comment:
@qianqianzhu I had to decrease the version of myst-parser to 0.14 because later version have the requirement of Sphinx 3.0 or greater. Can you confirm what versions of Sphinx an myst-parser you use? Maybe we can increase the version for Sphinx to 3.x instead?

Reply:
I can confirm that myst-parser 0.14 works fine for me. Thanks for the update.